src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let stringFromJoinedArray = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const itemContent
+        = this[i] === null || this[i] === undefined
+          ? ''
+          : this[i];
```

Comment:
I suggest using 'if' block here

Reply:
Is it for the sake of readability or why?